src/CMakeLists.txt

```diff
@@ -219,6 +219,12 @@ libcvc5_add_sources(
   proof/theory_proof_step_buffer.h
   proof/unsat_core.cpp
   proof/unsat_core.h
+  proof/alethe/alethe_post_processor.cpp
+  proof/alethe/alethe_post_processor.h
+  proof/alethe/alethe_proof_rule.cpp
+  proof/alethe/alethe_proof_rule.h
+  proof/alethe/alethe_printer.cpp
+  proof/alethe/alethe_printer.h
```

Comment:
remove this for now. You can add the files to compile after everything is merged in

Reply:
Sorry this should not have been added.